oauth2client/contrib/gce.py

```diff
@@ -134,8 +133,8 @@ def _refresh(self, http_request):
             self._retrieve_info(http_request)
             self.access_token, self.token_expiry = _metadata.get_token(
                 http_request, service_account=self.service_account_email)
-        except httplib2.HttpLib2Error as e:
-            raise client.HttpAccessTokenRefreshError(str(e))
+        except transport.transport_error() as err:
```

Comment:
@jonparrott @elibixby It seems this exception is only relevant here because of the usage of `httplib2.HttpLib2Error` in `oauth2client.contrib._metadata.get()`.

Is this accurate? Could a different, library-agnostic exception be used instead?